theme/templates/donate.html

```diff
@@ -0,0 +1,140 @@
+{% extends 'base.html' %}
+{% block title %} {% trans %}Donate{% endtrans %} | {{ super() }} {% endblock title %}
+
+{% block main %}
+<section class="app-banner">
+  <div class="container">
+    <div class="text-center section-space">
+      <h1 class="white-text-color app-banner-heading heading">{% trans %}Donate to the Tryton Software Foundation{% endtrans %}</h1>
+      <p class="white-text-color">{% trans %}Help the Tryton project to achieve its goal which ultimately brings a better free software business solution for companies all over the world.{% endtrans %}
+      </p>
+    </div>
+  </div>
+</section>
+<section>
+  <div class="container">
+    <ol class="breadcrumb breadcrumb-background">
+      <li><a class="color-black" href="{{ SITE_ROOT }}index.html">{% trans %}Home{% endtrans %}</a></li>
+      <li class="color-black">{% trans %}Donate{% endtrans %}</li>
+    </ol>
+  </div>
+</section>
+<section class="section-space">
+  <div class="container">
+    <div class="col-md-12 grid-spaces">
+      <p class="text-center donate-heading">{% trans %}No matter they're big or small, all of them benefit from a powerful application that is free from any vendor lock ins.if you're already using tryton,please consider contributing to the project or support the Foundation so the project remains vibrant and continues growing.We're happy to accept direct donation via:{% endtrans %}</p>
```

Comment:
First letter capital after fullstop ``.`` and also space after comma ``,`` and fullstop ``.``